### PR TITLE
docs(contributing): Updates to contribution guide

### PR DESCRIPTION
**Pull request by mffap**
- Fix heading
- De-duplicate PR section
- More info on PR section

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,15 +32,63 @@ Follow [@zitadel](https://twitter.com/zitadel) on twitter
...
+
+`git checkout -b my-fix-branch main`
+
+3. Make your changes following the [guidelines](#contribute) in this guide. Make sure that all tests pass.
+
+4. Commit the changes on the new branch
+
+`git commit --all`
+
+5. [Merge](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) the latest commit of the `main`-branch
```
Comment (fforootd) [inline CONTRIBUTING.md:54]:
```suggestion
5. [Merge](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) the latest commit of the `v2-alpha`-branch
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,15 +32,63 @@ Follow [@zitadel](https://twitter.com/zitadel) on twitter
...
+
+6. Push the changes to your branch on Github
+
+`git push origin my-fix-branch`
+
+7. Use [Semantic Release commit messages](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#type) to simplify creation of release notes. In the title of the pull request [correct tagging](#commit-messages) is required and will be requested by the reviewers.
+
+8.  On Github, [send a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to `zitadel:main`. Request review from one of the maintainers.
```
Comment (fforootd) [inline CONTRIBUTING.md:62]:
```suggestion
8.  On Github, [send a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to `zitadel:v2-alpha`. Request review from one of the maintainers.
```

**Review by fforootd — COMMENTED** [review]
Something is strange with this PR

Do you include the PR #3700 by intention?

File: CONTRIBUTING.md (hunk 1 above)
Comment (mffap) [reply CONTRIBUTING.md:54]: I've added the warning on top. Do you think that's not enough?

Comment (fforootd) [reply CONTRIBUTING.md:54]: I am torn with this one, because most of the times user simply copie stuff 😁 

But its not a deal breaker to merge this

**Comment by mffap** [comment]
issue with base. will be replaced.

**Pull request closed without merging**